Parse/SMTLIB2.cpp

```diff
@@ -160,6 +162,19 @@ void SMTLIB2::readBenchmark(LExprList* bench)
       continue;
     }
 
+    // TODO(mhajdu): is it possible that codatatypes are
```

Comment:
I couldn't find 'declare-codatatype(s)' in the 2.6 documentation but the plural keyword is parsed [here](https://github.com/vprover/vampire/pull/211/files#diff-0f0f75c3f4c1c2f5fc5b277a47a0393152567d1a4f14eb32e99564fd598c2dcfR189). So where does this come from?

Reply:
This is not an official SMTLIB keyword, but co-inductive data types are a fairly natural extension. The syntax of the declaration would be exactly similar, only the semantics would change (in particular w.r.t. circular terms). CVC4 supports them, and there was some work for Vampire but it never made it to master, so you can safely ignore this keyword.

Reply:
Thanks for the clarification. Then, I won't parse the 'co' counterpart now but leave the last argument `codatatype` of `readDeclareDatatype` there to be consistent with the plural `readDeclareDatatypes`.

Reply:
Wasn't codatatypes a pull request at some ancient time?

Reply:
Yes there is a 3 year old pull request that provides proper handling of codatatypes, plus some minor improvements for datatypes too (e.g., proper handling of cyclic terms in nested datatypes). The code has diverged a bit since then obviously, but if anybody is interested in reviewing that pull request, I could find some time to help.

Reply:
I removed the `codatatype` argument for now.